src/app/modules/backbone/components/floating-action/floating-action.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="floating-action-background">
```

Comment:
why the container div? Can be removed

You only a button that uses fixed position

Reply:
Ah ok, I think I misunderstood—I added in the background so that I could dim the rest of the page when the button appears. I will remove it.